.github/workflows/CI.yml

```diff
@@ -0,0 +1,202 @@
+name: CI
+# Run on master, tags, or any pull request
+on:
+  schedule:
+    - cron: '0 2 * * *'  # Daily at 2 AM UTC (8 PM CST)
+  push:
+    branches: [master]
+    tags: ["*"]
+  pull_request:
+env:
+  PGDATABASE: postgres
+  PGUSER: postgres
+  PGPASSWORD: root
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - PostgreSQL ${{ matrix.postgresql-version }} - ${{ matrix.os }} - ${{ matrix.arch }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - "1"  # Latest Release
+        postgresql-version:
+          - latest
+        os:
+          - ubuntu-latest
+          - macOS-latest
+          - windows-latest
+        arch:
+          - x64
+          - x86
+        exclude:
+          # Don't test 32-bit on macOS
+          - os: macOS-latest
+            arch: x86
+        include:
+          # Add a 1.0 job just to make sure we still support it
+          - os: ubuntu-latest
+            version: "1.0"
+            arch: x64
+            postgresql-version: latest
+          # Add a 1.3 job because that's what Invenia actually uses
+          - os: ubuntu-latest
+            version: 1.3
+            arch: x64
+            postgresql-version: latest
+          # Add older PostgreSQL Versions
+          - os: ubuntu-latest
+            version: 1
+            arch: x64
+            postgresql-version: '12'
+          # Add older supported PostgreSQL Versions
```

Comment:
```suggestion
            postgresql-version: '12'
```

Reply:
ah yeah I wasn't sure if that slightly different message was intentional or not